app/forms/manage_users_form.rb

```diff
@@ -35,4 +33,13 @@ def user_emails_are_valid_email_addresses
       errors.add(:base, "#{email} is not a valid email address")
     end
   end
+
+  def existing_users
+    User.where('lower(email) IN (?)', user_emails.map(&:downcase)).to_a
```

Comment:
Messed up my review comment.

One think we could probably do here is downcase `user_emails` on submit:

```ruby
def submit(user_emails)
  @user_emails = user_emails.map(&:downcase)
end
```

I'll leave whether to do that up to you.

Reply:
What if we migrated the table to be `citext`? https://www.postgresql.org/docs/9.1/citext.html

Reply:
Yes! That cleans it up considerably. I wrote this prior to discovering that devise was already downcasing emails on save.

Reply:
@zachmargolis with devise already downcasing the emails, I don't want to mess with the existing schema/index. I tried to play with it, but couldn't quickly figure out how to get a rails migration to recognize the `citext` type.

Reply:
Ok as long as Devise is doing essentially the same thing, it's good then